spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -583,9 +585,18 @@ def setup_menu(self, minmax):
                                                icon=ima.icon('filesave'),
                                                triggered=self.save_array)
         self.save_array_action.setVisible(False)
-        self.insert_action = create_action(self, _("Insert"),
-                                           icon=ima.icon('insert'),
-                                           triggered=self.insert_item)
+        self.insert_action = create_action(
+                      self, _("Insert"),
+                      icon=ima.icon('insert'),
+                      triggered=lambda: self.insert_item(below=False))

```

Comment:
Please rewrite this code as follows:

```python
self.insert_action = create_action(
    self, _("Insert"),
    icon=ima.icon('insert'),
    triggered=lambda: self.insert_item(below=False)
)
```

And do the same for the actions below this one.

Reply:
Thank you @ccordoba12 for your inputs. It seems that these changes triggered more fails in automated tests.
Let me know if you want further changes.

Reply:
@ccordoba12 I did the changes that you mentioned, but 5 tests failed. Is it something that i need to worry about?

Reply:
I'm not sure. For now please merge with the latest `4.x` branch to get the fixes to some of our tests.